src/WebServer_JSON.ino

```diff
@@ -3,7 +3,41 @@
 #include "src/Globals/Plugins.h"
 #include "StringProviderTypes.h"
 
+// ********************************************************************************
+// Web Interface get RAW value from task
+// ********************************************************************************
+void handle_rawval()
+{
+  String htmlData;
+  const taskIndex_t taskNr    = getFormItemInt(F("tasknr"), INVALID_TASK_INDEX);
+  const bool taskValid = validTaskIndex(taskNr);
+  if (!taskValid)
+  {
+    htmlData += F("ERROR: TaskNr not valid!\n");
+  }
+  const taskVarIndex_t valNr    = getFormItemInt(F("valnr"), INVALID_TASKVAR_INDEX);
+  const bool valueNumberValid = validTaskVarIndex(valNr);
+  if (!valueNumberValid)
+  {
+    htmlData += F("ERROR: ValueId not valid!\n");
+  }
 
+  TXBuffer.startJsonStream();
```

Comment:
Why start a JSON stream here and not output JSON formatted values?

Reply:
Good question, as i'm new to ESPEasy...Should i invoke some start()/end() function before sending plain HTTP text?

Reply:
Well, if you need a different mime type then we may need to add a different startStream function.
So I was just wondering if we now violate some standards here.

Reply:
It need to stay as it is, because without it browser won't recognize newline characters in plain text.

Reply:
That's OK.
Like I said, I just wondered whether we're violating some standards now.